tensorboard/components/tf_backend/test/backendTests.ts

```diff
@@ -81,6 +81,23 @@ describe('backend tests', () => {
       assert.equal(r.runs(), '/foo/runs');
     });
 
+    describe('default router', () => {
+      beforeEach(function() {
+        this.router = getDefaultRouter();
```

Comment:
Why assign to `this.router` and introduce this state instead of just
using

```js
        assert.equal(
            getDefaultRouter().runs(),
            `${window.location.pathname}/data/runs`
        );
```

in the one test case that actually uses this?